eo-runtime/src/test/eo/org/eolang/bytes-tests.eo

```diff
@@ -374,11 +374,11 @@
       $.equal-to
         "hello world"
 
-[] > bytes-hashes-equals
-  assert-that > @
-    05-5E-78.as-hash
-    $.equal-to
-      05-5E-78.as-hash
+# [] > bytes-hashes-equals
```

Comment:
@andreoss do we need this commented code?

Reply:
@mximp There is a todo to reimplement `as-hash` and enable these two tests

Reply:
@andreoss sorry, missed that was related. got it.